src/main/resources/local.properties.template

```diff
@@ -0,0 +1,3 @@
+# create a file here named local.properties and put the actual passwords there
+spring.datasource.password=example-root-password
```

Comment:
la password qui è sicura? non dovrebbero sempre essere posizionate in un .env

Reply:
questo è il file di esempio committato per riferimento dei developer! per far funzionare la stack dei servizi DEVI avere un file chiamato local.properties che contenga i valori veri.

comunque è una soluzione (imperfetta) che utilizziamo solo durante il development! in "produzione" usiamo spring vault + docker/kube secrets.

anche questo file 
https://github.com/GruppOne/stalker-server/blob/d5af28058869c2274cfd73da15c1edc7934d02fd/.env.template#L1-L7

funziona in maniera simile. devo ancora trovare un modo per "unirli" in modo che vada bene sia a spring che a docker